specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/kustoPool.json

```diff
@@ -3914,6 +4093,34 @@
       },
       "description": "The result returned from a check name availability request."
     },
+    "DatabaseInviteFollowerRequest": {
+      "type": "object",
+      "properties": {
+        "inviteeEmail": {
+          "type": "string",
+          "description": "The email of the invited user for which the follower invitation is generated."
```

Comment:
Add resource name restrictions for emails. [https://github.com/Azure/azure-openapi-validator/blob/main/docs/resource-name-restriction.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/resource-name-restriction.md)

Reply:
done